src/locales/cy/translation.json

```diff
@@ -855,8 +855,11 @@
         "subHeading": "Os ydych eisoes wedi bod i Swyddfa’r Post",
         "paragraph3": "Mae eich gwiriad hunaniaeth yn dal i gael ei brosesu.",
         "paragraph4": "Byddwch yn cael e-bost pan fydd canlyniad i’ch gwiriad hunaniaeth – fel arfer o fewn diwrnod o fynd i Swyddfa’r Post.",
-        "subHeading2": "Os ydych angen help",
-        "paragraph5": "Cysylltwch â ni os:",
+        "subHeading2": "Os ydych am brofi eich hunaniaeth mewn ffordd arall",
+        "paragraph5": "Os nad ydych bellach am brofi eich hunaniaeth mewn Swyddfa Bost, gallwch <a class=\"govuk-link\" href=\"/ipv/page/pyi-f2f-delete-details\">ddileu eich manylion a cheisio profi eich hunaniaeth mewn ffordd arall</a>.",
```

Comment:
You need to change the welsh one as well 🏴󠁧󠁢󠁷󠁬󠁳󠁿

Reply:
ah thanks!